### PR TITLE
added packages/pardi/pardi.2.0.4

### PR DESCRIPTION
**Pull request by UnixJunkie**
compatible with latest parany dep

### CODE REVIEW

**Comment by UnixJunkie** [comment]
supersedes https://github.com/ocaml/opam-repository/pull/16129

**Comment by kit-ty-kate** [comment]
It seems that the name of the `zmq` findlib package changed between version 4.0 and 5.0 (`ZMQ` vs `zmq`):
```
#=== ERROR while compiling pardi.2.0.4 ========================================#
# context              2.0.6 | linux/x86_64 | ocaml-base-compiler.4.06.1 | pinned(https://github.com/UnixJunkie/pardi/archive/v2.0.4.tar.gz)
# path                 ~/.opam/4.06/.opam-switch/build/pardi.2.0.4
# command              ~/.opam/4.06/bin/dune build -p pardi -j 72
# exit-code            1
# env-file             ~/.opam/log/pardi-23-b392a7.env
# output-file          ~/.opam/log/pardi-23-b392a7.out
### output ###
# File "src/dune", line 11, characters 57-60:
# 11 |   (libraries batteries dolog parany minicli str sec_chan zmq))
#                                                               ^^^
# Error: Library "zmq" not found.
# Hint: try: dune external-lib-deps --missing -p pardi -j 72 @install
```

File: packages/pardi/pardi.2.0.4/opam [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+bug-reports: "https://github.com/UnixJunkie/pardi/issues"
+dev-repo: "git+https://github.com/UnixJunkie/pardi.git"
+license: "GPL-1.0-or-later"
+build: ["dune" "build" "-p" name "-j" jobs]
+depends: [
+  "dune" {>= "1.11"}
+  "batteries"
+  "dolog" {>= "4.0.0"}
+  "parany" {>= "6.0.0"}
+  "minicli"
```
Comment (kit-ty-kate) [inline packages/pardi/pardi.2.0.4/opam:14]: Hasn't minicli also changed it interface name completely in `minicli.5.0.0` or something?

**Comment by kit-ty-kate** [comment]
could you also return the fix upstream?

File: packages/pardi/pardi.2.0.4/opam (hunk 1 above)
Comment (kit-ty-kate) [reply packages/pardi/pardi.2.0.4/opam:14]: I just tried locally and it failed with minicli.4.0.0. I'm fixing that.

**Comment by kit-ty-kate** [comment]
All good. Thanks! Could you return the fixes upstream?

**Comment by UnixJunkie** [comment]
thanks, I updated the opam file to include your fixes

**Pull request merged**